joplin/base/signals.py

```diff
@@ -33,7 +33,7 @@ def generate_responsive_images(sender, **kwargs):
         image.get_rendition(f'width-{width}')
 
 
-def create_build_aws(instance, publish=False, request=None):
+def create_build_aws(content_type, instance, publish_action='edited', request=None):
```

Comment:
Is `content_type` only used for logging in our slack messages?

Reply:
yes, this is only for slack messages, figured it'd be more clear this way considering we have "Recycle and Reuse drop off center" as a map, location, and contact